renderer/components/exports/export.js

```diff
@@ -52,16 +53,25 @@ export default class Export extends React.Component {
     const {menu} = this.state;
 
     const cancelable = status === 'waiting' || status === 'processing';
-
+    const fileNameClassName = classNames({
+      title: true,
+      'title-color': !cancelable,
+      'title-color-disabled': cancelable
```

Comment:
I don't think we need 3 classes, since the two are mutually exclusive. Could be:
```
{
	title: true
	disabled: cancelable
}
```

Then, add the default title color to the `title` class and the disabled color on the `.title.disabled` class

Reply:
Also, not sure `cancelable` is the right check here (might be in error state, so cancelable is `false`, but this should still be disabled). I think we need a new variable called `isActionable`. The value should be set to the same thing as line 31:

`const isActionable = filePath && !disableOutputActions`

Might even be worth abstracting it into a getter on the class, so we can check as `this.isActionable`

```
get isActionable() {
	const {filePath, disableOutputActions} = this.props;
	return filePath && !disableOutputActions;
}
```

And then we can use it here, in the `openFile` method and the `onDragStart` as well (which is missing the `filePath` check but it should probably be there)

Reply:
Thanks, this is a much better option. Canceled titles also stay greyed out now.